### ISSUE TITLE
Attach individual jars to GitHub release instead of zip file

### ISSUE THREAD

**Issue by falko** [camunda-community-hub/community-action-maven-release#38]
My build for https://github.com/camunda-community-hub/zeebe-embedded-job-worker produces four different jar files:

1. zeebe-embedded-job-worker-v0.0.6.jar
2. zeebe-embedded-job-worker-v0.0.6-jar-with-dependencies.jar
3. zeebe-embedded-job-worker-v0.0.6-javadoc.jar
4. zeebe-embedded-job-worker-v0.0.6-sources.jar

However, the workflow attaches a single zip file to the GitHub release, which contains all four of them in a deep directory called `/target/nexus-staging/deferred/org/camunda/community/extension/zeebe/exporter/jobworker/zeebe-embedded-job-worker/v0.0.6/`

How can I get the individual jar files attached to the release instead?

I need to be able to easily download the `zeebe-embedded-job-worker-v0.0.6.jar` from within an init container for the Zeebe broker.

In other words, I don't care too much about Maven Central as long as I have an automated way to get the jar built and served online for downloading.

**Comment by falko** [comment]
Okay, as a workaround I found the jar in Artifactory: https://github.com/camunda-community-hub/zeebe-embedded-job-worker/releases/download/0.0.7/zeebe-embedded-job-worker-0.0.7.jar

**Comment by berndruecker** [comment]
Would you agree to close this issue? I think getting the artifacts from Artifactory is actually a good way - not a too bad workaround? Reopen if you think otherwise!
